client/X11/xf_window.c

```diff
@@ -935,9 +934,16 @@ void xf_MoveWindow(xfContext* xfc, xfAppWindow* appWindow, int x, int y,
 	appWindow->height = height;
 
 	if (resize)
+	{
+		DEBUG_X11("setting new internal & X geometry: %dx%d+%d+%d\tshowState=%d", width, height, x, y,
```

Comment:
Please use the appropriate format string macros

Reply:
For variables declared with no bit size (e.g., `int x`), should I use `PRId32`? I didn't see a macro that uses the right size for just plain `int`.